stable/kommander/values.yaml

```diff
@@ -18,6 +18,10 @@ kommander-karma:
   kommanderServiceAccount: kommander-kubeaddons
   kommanderKarmaConfigMap: kommander-kubeaddons-config
 
+kubecost:
+  enabled: false
+  thanosAddress: "kommander-kubeaddons-kubecost-kubecost-kubeaddons-prometheus-se.kubecost.svc.cluster.local:10901"
```

Comment:
that super long address is maxed at 63 chars... could there be a potential nameclash problem on a restart?

Reply:
hopefully there are no other prometheus components that start with `se` 🙃 but yeah the name length is unfortunate, this is even after I renamed this subchart from `kommander-kubecost` to just `kubecost`. I think the other option is to override this dynamically-created name entirely and set our own name which I had tried out in my testing as well.

Reply:
We most likely can pick this up dynamically, can we check if that is possible before going this route of using a static value>